pkg/snapshot/snapshot.go

```diff
@@ -0,0 +1,99 @@
+package snapshot
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hyperledger/fabric-admin-sdk/internal/protoutil"
+	"github.com/hyperledger/fabric-admin-sdk/pkg/identity"
+	"github.com/hyperledger/fabric-protos-go-apiv2/common"
+	"github.com/hyperledger/fabric-protos-go-apiv2/peer"
+	"google.golang.org/grpc"
+	"google.golang.org/protobuf/proto"
+)
+
+// Client is a wrapper around snapshot client
+type Client struct {
+	snapshot peer.SnapshotClient
+	id       identity.SigningIdentity
+}
+
+func New(conn grpc.ClientConnInterface, id identity.SigningIdentity) *Client {
```

Comment:
This would be perfectly named if it was creating a Snapshot, since the package is called snapshot. It might be clearer (and more consistent with other packages in this module) if it names what it is creating. For example, NewPeer.